academy/ibc-dev/ibcapp_packet.md

```diff
@@ -0,0 +1,407 @@
+# Adding Packet and Acknowledgement data
```

Comment:
@tmsdkeys I changed "acknowledgement(s)" to "acknowledgment(s)" throughout the text but not in all content that is in code tags because I wasn't sure because we are using American English. Please check, if it can be changed in the code or has to remain in British English. For the latter, I would leave it as it is with my commits, in the first case, please change it throughout the section.

Reply:
In the code, it must be "acknowledgement". In a way, it seems logical to keep it consistent. On the other hand, I don't have any strong opinions as long as we keep it consistent and correct in the code and when referring to an piece of code in text like `OnAcknowledgementPacket`

Reply:
OK, that's the way I handled it. 😉